Tests/SentryTests/Networking/SentryReachabilityTests.m

```diff
@@ -69,41 +70,39 @@ - (void)testConnectivityRepresentations
 
 - (void)testMultipleReachabilityObservers
 {
-    NSLog(@"[Sentry] [TEST] creating observer A");
+    printf("[Sentry] [TEST] creating observer A\n");
     TestSentryReachabilityObserver *observerA = [[TestSentryReachabilityObserver alloc] init];
-    NSLog(@"[Sentry] [TEST] adding observer A as reachability observer");
+    printf("[Sentry] [TEST] adding observer A as reachability observer\n");
     [self.reachability addObserver:observerA];
 
-    NSLog(@"[Sentry] [TEST] throwaway reachability callback, setting to reachable");
+    printf("[Sentry] [TEST] throwaway reachability callback, setting to reachable\n");
     SentryConnectivityCallback(self.reachability.sentry_reachability_ref,
         kSCNetworkReachabilityFlagsReachable, nil); // ignored, as it's the first callback
-    NSLog(@"[Sentry] [TEST] reachability callback to set to intervention required");
-    SentryConnectivityCallback(self.reachability.sentry_reachability_ref,
-        kSCNetworkReachabilityFlagsInterventionRequired, nil);
+    printf("[Sentry] [TEST] reachability callback set to unreachable\n");
+    SentryConnectivityCallback(self.reachability.sentry_reachability_ref, 0, nil);
 
-    NSLog(@"[Sentry] [TEST] creating observer B");
+    printf("[Sentry] [TEST] creating observer B\n");
     TestSentryReachabilityObserver *observerB = [[TestSentryReachabilityObserver alloc] init];
-    NSLog(@"[Sentry] [TEST] adding observer B as reachability observer");
+    printf("[Sentry] [TEST] adding observer B as reachability observer\n");
     [self.reachability addObserver:observerB];
 
-    NSLog(@"[Sentry] [TEST] reachability callback to set to back to reachable");
+    printf("[Sentry] [TEST] reachability callback set back to reachable\n");
     SentryConnectivityCallback(
         self.reachability.sentry_reachability_ref, kSCNetworkReachabilityFlagsReachable, nil);
-    NSLog(@"[Sentry] [TEST] reachability callback to set to back to intervention required");
-    SentryConnectivityCallback(self.reachability.sentry_reachability_ref,
-        kSCNetworkReachabilityFlagsInterventionRequired, nil);
+    printf("[Sentry] [TEST] reachability callback set back to unreachable\n");
+    SentryConnectivityCallback(self.reachability.sentry_reachability_ref, 0, nil);
```

Comment:
I realized that `kSCNetworkReachabilityFlagsInterventionRequired` shouldn't even be detected by our reachability monitoring because we only return `YES` for "reachable" (and with UIKit, also "wwan"). Not sure "intervention required" should've ever worked, but trying it with 0 instead.